content/en/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md

```diff
@@ -75,6 +75,11 @@ In the above manifest, you use a [KubeSchedulerConfiguration](/docs/reference/sc
 to customize the behavior of your scheduler implementation. This configuration has been passed to
 the `kube-scheduler` during initialization with the `--config` option. The `my-scheduler-config` ConfigMap stores the configuration file. The Pod of the`my-scheduler` Deployment mounts the `my-scheduler-config` ConfigMap as a volume.
 
+Grants a Role  within a specific namespace. Example:
+ ```shell
+    kubectl create rolebinding -n kube-system my-scheduler-extension-apiserver-authentication-reader --role=extension-apiserver-authentication-reader --serviceaccount=kube-system:my-scheduler
```

Comment:
```suggestion
kubectl create rolebinding -n kube-system my-scheduler-extension-apiserver-authentication-reader --role=extension-apiserver-authentication-reader --serviceaccount=kube-system:my-scheduler
```

What about the ServiceAccount? Where is that created?

Reply:
Done

Reply:
@sftim ServiceAccount is created in kube-system namespace , please see here
https://kubernetes.io/docs/tasks/extend-kubernetes/configure-multiple-schedulers/#define-a-kubernetes-deployment-for-the-scheduler

Reply:
> $ kubectl get serviceaccount -n kube-system
> NAME                                 SECRETS   AGE
> attachdetach-controller              0         18d
> bootstrap-signer                     0         18d
> calico-kube-controllers              0         18d
> canal                                0         18d
> certificate-controller               0         18d
> clusterrole-aggregation-controller   0         18d
> coredns                              0         18d
> cronjob-controller                   0         18d
> daemon-set-controller                0         18d
> default                              0         18d
> deployment-controller                0         18d
> disruption-controller                0         18d
> endpoint-controller                  0         18d
> endpointslice-controller             0         18d
> endpointslicemirroring-controller    0         18d
> ephemeral-volume-controller          0         18d
> expand-controller                    0         18d
> generic-garbage-collector            0         18d
> horizontal-pod-autoscaler            0         18d
> job-controller                       0         18d
> kube-proxy                           0         18d
> my-scheduler                         0         4m32s

Please find the output here

Reply:
Hi @sftim Could you please let me know if i addressed your query properly , or if any other thing is left.